docs/deploy.md

```diff
@@ -0,0 +1,174 @@
+Deploying Mail for Good on AWS
+===================
+
+
+Creating the blank instance
+--------------
+
+Right after you log in AWS you will land on this page
+![](resources/deploy_images/1.png)
+
+
+Scroll down until you see the "build a solution" box like below and select "Launch a virtual machine"
+![](resources/deploy_images/2.png)
+
+Click on the "Get started" button in the "EC2 Instance" box.
+![](resources/deploy_images/3.png)
+
+Give a name to your instance. In the example, we will go with "Mail-for-Good"
+![](resources/deploy_images/4.png)
+
+Select Ubuntu as your operating system
+![](resources/deploy_images/5.png)
+
+Here the preselected type of instance is fine.(t2.micro instances are eligible to the free tier)
+![](resources/deploy_images/6.png)
+
+AWS will create a pair of keys to your instance. You will have to supply this file every time you will want to log into your instance. Right now AWS is just asking for a name. For the example, we will go with "MfG-private-key". For the example, we will also store this file in our home. /home/{your username}/ also written ~/
+![](resources/deploy_images/7.png)
+
+Now AWS is creating your instance.
+![](resources/deploy_images/8.png)
+
+You should see this once it's done.
+![](resources/deploy_images/9.png)
+
+
+Allow your instance to receive requests from your clients
+---------
+In the navbar at the top of your screen select "EC2" under the "Compute" category.
+![](resources/deploy_images/10.png)
+
+In the menu on the left of your screen, click on "Instances"
+![](resources/deploy_images/11.png)
+
+Then scroll the box containing the instances to its far right until you see "security Groups"
+![](resources/deploy_images/13.png)
+
+
+click on the security group of your instance and open tho "Inbound" tab
+![](resources/deploy_images/14.png)
+then "Edit"
+![](resources/deploy_images/15.png)
+Add the following rule
+type:custom tcp rule
+protocol: tcp
+port range:8080
+source: custom 0.0.0.0/0, ::/0
+
+
+
+Deploying Mail for Good on your instance
+---------
+### Log into the instance
```

Comment:
Is it also possible to add an existing SSH public key to this box? E.g. adding a client key at ~/.ssh/id_rsa.pub to the server's authorized_keys. That would remove the need for using another private key on the client & having to select it when using ssh.